native/mod_manager/mod_manager.c

```diff
@@ -1206,6 +1206,35 @@ static void mark_node_removed(nodeinfo_t *node)
         node->mess.num_remove_check = 0;
     }
 }
+static proxy_worker_shared *read_shared_by_node(request_rec *r, nodeinfo_t *node)
+{
+    void *sconf = r->server->module_config;
+    int i, port;
+    char *name = apr_pstrcat(r->pool, "balancer://", node->mess.balancer, NULL);
+    if (sscanf(node->mess.Port, "%u", &port)!=1)
+        return NULL; /* something is wrong */
+    proxy_server_conf *conf = (proxy_server_conf *)ap_get_module_config(sconf, &proxy_module);
+    proxy_balancer *balancer = (proxy_balancer *)conf->balancers->elts;
+    for (i = 0; i < conf->balancers->nelts; i++, balancer++) {
+        int j;
+        proxy_worker **workers;
+        if (strcmp(balancer->s->name, name))
+            continue;
+        workers = (proxy_worker **)balancer->workers->elts;
+        for (j = 0; j < balancer->workers->nelts; j++, workers++) {
+            proxy_worker *worker = *workers;
+            ap_log_error(APLOG_MARK, APLOG_DEBUG, 0, r->server, "Balancer %s worker %s, %s, %d",
```

Comment:
Please add `read_shared_by_node:` at the beginning of the debug message.

Reply:
fixed